lib/features/index.js

```diff
@@ -0,0 +1,6 @@
+// @ts-check
+const { FeatureManager, FeatureGrantType } = require('./manager');
+
+const features = new FeatureManager(['course:manual-grading']);
```

Comment:
TODO: empty this out before merging.

```suggestion
const features = new FeatureManager([]);
```